### DISCUSSION TITLE
About an issue with saleor dashboard 3.5.1

### DISCUSSION THREAD

**Discussion by Batgerel53** [saleor/saleor#10557] [Q&A]
Hello   

I am using saleor 3.5.1 main dashboard version 3.5.1. suddenly this error occurred. I don't know why. I am facing this problem for last 1 week.Please help me.  @karolkielecki @orzechdev @patrys  @maarcingebala 

![dashboard_error](https://user-images.githubusercontent.com/56902009/188543748-12b79bb0-1e2f-4f67-b8db-e9a22645fcdb.PNG)

**Comment by Batgerel53** [comment]
When I click on order details, this error appears. I have not made any changes to the code of the dashboard.

**Reply by timuric** [reply]
Which version of saleor are you using?

**Reply by Batgerel53** [reply]
@timuric  saleor 3.5.1

**Reply by Batgerel53** [reply]
I solved this problem. warehouse configured.
